app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -280,23 +277,25 @@ fun AssociationProfileEvents(context: Context) {
             modifier = Modifier.padding(horizontal = 28.dp).testTag("AssociationSeeMoreButton")) {
               Icon(Icons.AutoMirrored.Filled.ArrowForward, contentDescription = "See more")
               Spacer(Modifier.width(2.dp))
-              Text(getString(R.string.association_see_more))
+              Text(context.getString(R.string.association_see_more))
             }
       }
 }
 
 @Composable
 fun AssociationEventTitle() {
+  val context = LocalContext.current
```

Comment:
This val could be passed as an argument to avoid duplicate code (already in the AssociationProfileScaffold)